src/resources/triton_python_backend_utils.py

```diff
@@ -340,6 +342,28 @@ def set_max_batch_size(self, max_batch_size):
                 str(max_batch_size))
         else:
             self._model_config["max_batch_size"] = max_batch_size
+            if max_batch_size > 1:
+                self.set_dynamic_batching()
+
+    def set_dynamic_batching(self):
+        """Set dynamic_batching as the scheduler for the model if no scheduler is set.
+           If dynamic_batching is already set, then no action is taken and return success.
+        Raises
+        ------
+        ValueError
+            If the configuration has a scheduler set which is not dynamic_batching.
+        """
+        if "sequence_batching" in self._model_config.keys():
```

Comment:
Should we return an error if config had any other scheduling policy? I think we must just not attempt to add dynamic batching in such cases like in rest of the backends.

Reply:
https://github.com/triton-inference-server/tensorflow_backend/blob/main/src/tensorflow.cc#L1316-L1331

Reply:
fixed, along with the tests.

Reply:
Attempting to set dynamic_batching as the scheduler when one is already set (including `dynamic_batching`) will raise an error. [Please see this thread for more context](https://github.com/triton-inference-server/python_backend/pull/165#discussion_r891619472)